tests/samplers_tests/tpe_tests/test_sampler.py

```diff
@@ -869,3 +869,15 @@ def test_reseed_rng() -> None:
         sampler.reseed_rng()
         assert mock_object.call_count == 1
         assert original_seed != sampler._rng.seed
+
+
+def test_call_after_trial_of_independent_sampler() -> None:
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", optuna.exceptions.ExperimentalWarning)
+        sampler = TPESampler()
```

Comment:
`TPESampler` is not experimental.
```suggestion
```

Reply:
I'll remove that. Thank you for reviewing.